<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="BodyDategroupBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Body</string>
    </property>
    <widget class="QComboBox" name="SexComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="HeightLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
    <widget class="QLabel" name="SexLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sex</string>
     </property>
    </widget>
    <widget class="QLabel" name="AgeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>31</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Age</string>
     </property>
    </widget>
    <widget class="QLabel" name="WeightLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Weight</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="AgeTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="WeightTextEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="HeightTextEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="NameLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>270</y>
      <width>174</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_Button">
     <item>
      <widget class="QPushButton" name="LoadpushButton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Load avatar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CreatepushButton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Create avatar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="ExitPushButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>590</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>170</width>
      <height>260</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="RaceClassAliggroupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Player</string>
    </property>
    <widget class="QLabel" name="RaceLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Race</string>
     </property>
    </widget>
    <widget class="QLabel" name="SubRaceLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Subrace</string>
     </property>
    </widget>
    <widget class="QComboBox" name="RaceComboBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="SubRaceTextEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="ClassesLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Classes</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ClassesComboBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="AlignmentLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Alignment</string>
     </property>
    </widget>
    <widget class="QComboBox" name="AlignmentComboBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="StatsgroupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>341</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Stats</string>
    </property>
    <widget class="QLabel" name="Stats_res_label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Result stats</string>
     </property>
    </widget>
    <widget class="QLabel" name="Stats_label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Stats</string>
     </property>
    </widget>
    <widget class="QLabel" name="Race_mod_label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Race mod</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Cha_spinBox">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>350</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="Wis_spinBox">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>296</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Dex_label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>109</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Dex</string>
     </property>
    </widget>
    <widget class="QLabel" name="Int_label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>217</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Int</string>
     </property>
    </widget>
    <widget class="QLabel" name="Con_label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>163</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Con</string>
     </property>
    </widget>
    <widget class="QLabel" name="Cha_label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>325</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cha</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Con_spinBox">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>188</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="Dex_spinBox">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>134</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="Str_spinBox">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>80</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="Int_spinBox">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>242</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Wis_label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>271</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Wis</string>
     </property>
    </widget>
    <widget class="QLabel" name="Str_label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>55</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Str</string>
     </property>
    </widget>
    <widget class="QLabel" name="custom_label_6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>350</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="custom_label_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>242</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="custom_label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="custom_label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>188</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="custom_label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>134</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="custom_label_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>296</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="race_mod_cha_label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>350</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>cha_race_mod</string>
     </property>
    </widget>
    <widget class="QLabel" name="race_mod_con_label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>188</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>con_race_mod</string>
     </property>
    </widget>
    <widget class="QLabel" name="race_mod_str_label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>str_race_mod</string>
     </property>
    </widget>
    <widget class="QLabel" name="race_mod_int_label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>242</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>int_race_mod</string>
     </property>
    </widget>
    <widget class="QLabel" name="race_mod_wis_label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>296</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>wis_race_mod</string>
     </property>
    </widget>
    <widget class="QLabel" name="race_mod_dex_label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>134</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>dex_race_mod</string>
     </property>
    </widget>
    <widget class="QLabel" name="custom_label_12">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>350</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QLabel" name="custom_label_11">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>296</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QLabel" name="custom_label_8">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>134</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QLabel" name="custom_label_10">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>242</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QLabel" name="custom_label_7">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QLabel" name="custom_label_9">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>188</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QLabel" name="cha_stats_label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>350</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>cha_stats</string>
     </property>
    </widget>
    <widget class="QLabel" name="str_stats_label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>str_stats</string>
     </property>
    </widget>
    <widget class="QLabel" name="con_stats_label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>188</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>con_stats</string>
     </property>
    </widget>
    <widget class="QLabel" name="int_stats_label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>242</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>int_stats</string>
     </property>
    </widget>
    <widget class="QLabel" name="wis_stats_label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>296</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>wis_stats</string>
     </property>
    </widget>
    <widget class="QLabel" name="dex_stats_label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>134</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>dex_stats</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="NameTextEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
